data/json/recipes/armor/head.json

```
@@ -610,13 +610,8 @@
     "using": [ [ "sewing_standard", 65 ], [ "soldering_standard", 8 ] ],
     "components": [
       [ [ "hat_ball", 1 ], [ "hat_boonie", 1 ] ],
-      [
-        [ "mask_filter", 1 ],
-        [ "mask_gas", 1 ],
-        [ "mask_survivor", 1 ],
-        [ "mask_halfsurvivor", 1 ],
-        [ "mask_hsurvivor", 1 ]
-      ],
+      [ [ "mask_filter", 1 ] ],
+      [ [ "mask_gas", 1 ], [ "mask_survivor", 1 ], [ "mask_halfsurvivor", 1 ], [ "mask_hsurvivor", 1 ] ],
```

As noted, filter mask only has 7 EP when provided with filters, so no real reason to inject it into this recipe as a separate component. Also since we're allowing half-masks which inherit from light masks, added that to the material list in this suggestion.
```suggestion
      [
        [ "mask_gas", 1 ],
        [ "mask_survivor", 1 ],
        [ "mask_lsurvivor", 1 ],
        [ "mask_halfsurvivor", 1 ],
        [ "mask_hsurvivor", 1 ]
      ]
```